shuup/xtheme/models.py

```diff
@@ -174,6 +179,98 @@ def __str__(self):
         return _("Theme configuration for %s") % self.theme_identifier
 
 
+@python_2_unicode_compatible
+class Font(models.Model):
```

Comment:
I guess that we can pretty much deprecate `shuup-typography` and use this instead. It would work way better with this inside Xtheme and making xtheme knows how to do it.